CMakeLists.txt

```diff
@@ -20,8 +20,8 @@ project(cfsd18-perception-detectcone)
 ################################################################################
 # Defining the relevant versions of OpenDLV Standard Message Set and libcluon.
```

Comment:
I usually add: 

find_program(CCACHE_FOUND` ccache)

if(CCACHE_FOUND)
    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
    set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
endif(CCACHE_FOUND)

to use ccache on the machines where it is installed. It often speeds up local compilation when you want to test things out.